test/ng/testabilitySpec.js

```diff
@@ -188,11 +188,36 @@ describe('$$testability', function() {
   });
 
   describe('waiting for stability', function() {
-    it('should process callbacks immediately with no outstanding requests',
-      inject(function($$testability) {
+    it('should process callbacks without waiting if there are no outstanding requests',
+      inject(function($$testability, $timeout) {
         var callback = jasmine.createSpy('callback');
-        $$testability.whenStable(callback);
-        expect(callback).toHaveBeenCalled();
+        runs(function() {
```

Comment:
Since unit tests run synchronously, you shouldn't need async tests (using `runs`/`waitsFor`). Have you tried without ?
(BTW, note that we have updated to Jasmine v2 now, so these tests won't run anymore (since `runs`/`waitsFor` is a Jasmine 1.x thing). But you shouldn't need them anyway (hopefully)...)